cpp_linter/run.py

```diff
@@ -451,9 +445,31 @@ def capture_clang_tools_output(
         arguments.
     """
 
+    def show_tool_version_output(cmd: str):  # show version output for executable used
+        version_out = subprocess.run(
+            [cmd, "--version"], capture_output=True, check=True
+        )
+        logger.info("%s --version\n%s", cmd, indent(version_out.stdout.decode(), "\t"))
```

Comment:
I just want to point out that I'm relying on clang-tools-pip to make sure the specified version is the actual version getting used. This log output here is just so we (& users) can manually check that the version specified is being used.